pkg/apis/pipeline/v1alpha1/run_types.go

```diff
@@ -47,7 +47,7 @@ type EmbeddedRunSpec struct {
 // RunSpec defines the desired state of Run
 type RunSpec struct {
 	// +optional
-	Ref *TaskRef `json:"ref,omitempty"`
+	Ref *v1beta1.TaskRef `json:"ref,omitempty"`
```

Comment:
Any change to the `TaskRef` in beta will impact `Runs` in alpha now.
I guess that should be ok `Run` didn't have its own `TaskRef` definition before anyways.

Reply:
Again, already the case: https://github.com/tektoncd/pipeline/blob/13a3c5d9612af58382b0fe31622ec0468f311d88/pkg/apis/pipeline/v1alpha1/pipeline_types.go#L226 =)